src/v/storage/segment_utils.cc

```diff
@@ -491,4 +493,17 @@ std::filesystem::path compacted_index_path(std::filesystem::path segment_path) {
     return segment_path.replace_extension(".compaction_index");
 }
 
+size_t jitter_segment_size(size_t sz, jitter_percents jitter_percents) {
+    vassert(
+      jitter_percents >= 0 || jitter_percents <= 100,
+      "jitter percents should be in range [0,100]. Requested {}",
+      jitter_percents);
+    // multiply by 10 to increase resolution
+    auto p = jitter_percents() * 10;
+    auto jit = random_generators::get_int<int64_t>(-p, p);
+
+    int64_t jitter = jit * static_cast<int64_t>(sz) / 1000;
```

Comment:
did you intend to divide by `100` since `jit` is a percentage?

Reply:
the `1000` is correct, I multiply by `10` to increase resolution to `1/10` of a percent

Reply:
got it. i thought `* 10` was to increase the spread. makes sense.